src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const arr of actions) {
+    switch (arr['type']) {
```

Comment:
```suggestion
    switch (arr.type) {
```

Reply:
you need square brackets only when you don't know name of property(when you use variables for that)